test/test_nn.py

```diff
@@ -2563,12 +2563,15 @@ def test_contig_wrong_stride_cudnn(self):
         F.conv2d(x, torch.randn(1, 16, 1, 1, device="cuda"))
 
     def test_embedding_bag(self):
-        self._test_EmbeddingBag(False, 'sum', False)
-        self._test_EmbeddingBag(False, 'mean', False)
-        self._test_EmbeddingBag(False, 'max', False)
-
-        self._test_EmbeddingBag(False, 'sum', True)
-        self._test_EmbeddingBag(False, 'mean', True)
+        for dtype in [torch.double, torch.float]:
+            # TODO: figure out why backward on float breaks
```

Comment:
I don't think it's "broken" on backward; it's just that the precision on the test needs to be adjusted because the gradients are so large. If this hypothesis is true, then test_embedding_bag should fail for `mode=sum` but not fail for `mode={max,mean}`.

Reply:
@zou3519 IME it fails for all off sum, max, and mean

Reply:
What is the magnitude of the gradient for "max" and "mean"? If it is on the order of magnitude of 1 then there are definitely precision issues, then